src/views/CreatePage.vue

```diff
@@ -361,5 +388,65 @@ div(class="m-auto border-4 border-orange-100 max-w-screen-xl my-4")
         //- )
         v-transformer(ref="transformer" :config="configShapeTransformer")
 div(class="container")
-  ToolBar(:stage="stage" :stage-parent-div="stageParentDiv" :save-canvas="saveCanvas")
+  ToolBar(:stage="stage")
+
+input(id="reset-modal" type="checkbox" className="modal-toggle")
+div(className="modal")
+  div(className="modal-box max-w-none w-auto")
+    h3(className="font-bold text-2xl") キャンバスをリセットしてよろしいですか？
+    div(class="flex justify-end")
+      div(className="modal-action mr-3")
+        label(htmlFor="reset-modal" className="btn w-36") Cancel
+      div(className="modal-action" @click="pointLeaderOpen = false")
+        label(htmlFor="reset-modal" className="btn w-36 bg-red-500 border-none hover:bg-red-600" @click="useStoreCanvas().resetCanvas") OK
```

Comment:
```suggestion
      div(className="modal-action mr-3" @click="pointLeaderOpen = false")
        label(htmlFor="reset-modal" className="btn w-36") Cancel
      div(className="modal-action" @click="pointLeaderOpen = false")
        label(htmlFor="reset-modal" className="btn w-36 bg-red-500 border-none hover:bg-red-600" @click="useStoreCanvas().resetCanvas") OK
```
キャンバスリセットをキャンセルした時、３点リーダーは一度閉じずに、そのままにしますか？
自分だったら、モーダルが閉じて一連の流れが終わったということで、非表示にするかなと思ったのですが。

Reply:
そこ正直悩んでます
興味で押したのならリーダーも消えた方が良いですが、
本当は別のを選択したかったのに間違えて押したら二度手間になるのでリーダーは閉じませんでした

Reply:
別の選択と言っても、今は共有の切り替えしかないので、正直そこまでの配慮がいるか微妙なところですね。
一度消そうと思ったけど、やっぱりキャンセルして作業に戻ろうと思った人からしたら、消えないと違和感あると思いました。

Reply:
そうですね。まあそこまで配慮しなくて良いか
キャンセルした時、３点リーダー消えるように修正します